.openpublishing.redirection.json

```diff
@@ -895,3444 +895,3444 @@
         "redirect_url": "/archive/microsoft-edge/legacy/developer",
         "redirect_document_id": false
       },
       {
-        "source_path": "microsoft-edge/edgehtml/hosting/chakra-hosting/jscreateexternalobject-function",
+        "source_path": "./microsoft-edge/edgehtml/hosting/chakra-hosting/jscreateexternalobject-function",
         "redirect_url": "/archive/microsoft-edge/legacy/developer",
         "redirect_document_id": false
       },
       {
-        "source_path": "microsoft-edge/edgehtml/hosting/chakra-hosting/jscreatefunction-function",
+        "source_path": "./microsoft-edge/edgehtml/hosting/chakra-hosting/jscreatefunction-function",
         "redirect_url": "/archive/microsoft-edge/legacy/developer",
         "redirect_document_id": false
       },
       {
-        "source_path": "microsoft-edge/edgehtml/hosting/chakra-hosting/jscreatenamedfunction-function",
+        "source_path": "./microsoft-edge/edgehtml/hosting/chakra-hosting/jscreatenamedfunction-function",
         "redirect_url": "/archive/microsoft-edge/legacy/developer",
         "redirect_document_id": false
       },
       {
-        "source_path": "microsoft-edge/edgehtml/hosting/chakra-hosting/jscreateobject-function",
+        "source_path": "./microsoft-edge/edgehtml/hosting/chakra-hosting/jscreateobject-function",
         "redirect_url": "/archive/microsoft-edge/legacy/developer",
         "redirect_document_id": false
       },
       {
-        "source_path": "microsoft-edge/edgehtml/hosting/chakra-hosting/jscreaterangeerror-function",
+        "source_path": "./microsoft-edge/edgehtml/hosting/chakra-hosting/jscreaterangeerror-function",
         "redirect_url": "/archive/microsoft-edge/legacy/developer",
         "redirect_document_id": false
       },
       {
-        "source_path": "microsoft-edge/edgehtml/hosting/chakra-hosting/jscreatereferenceerror-function",
+        "source_path": "./microsoft-edge/edgehtml/hosting/chakra-hosting/jscreatereferenceerror-function",
         "redirect_url": "/archive/microsoft-edge/legacy/developer",
         "redirect_document_id": false
       },
       {
-        "source_path": "microsoft-edge/edgehtml/hosting/chakra-hosting/jscreateruntime-function",
+        "source_path": "./microsoft-edge/edgehtml/hosting/chakra-hosting/jscreateruntime-function",
         "redirect_url": "/archive/microsoft-edge/legacy/developer",
         "redirect_document_id": false
       },
       {
-        "source_path": "microsoft-edge/edgehtml/hosting/chakra-hosting/jscreatesymbol-function",
+        "source_path": "./microsoft-edge/edgehtml/hosting/chakra-hosting/jscreatesymbol-function",
         "redirect_url": "/archive/microsoft-edge/legacy/developer",
         "redirect_document_id": false
       },
       {
-        "source_path": "microsoft-edge/edgehtml/hosting/chakra-hosting/jscreatesyntaxerror-function",
+        "source_path": "./microsoft-edge/edgehtml/hosting/chakra-hosting/jscreatesyntaxerror-function",
```

Comment:
no .md here??
@captainbrosset

Reply:
maybe test
https://learn.microsoft.com/microsoft-edge/hosting/chakra-hosting/jscreatesyntaxerror-function
end up at:
https://learn.microsoft.com/en-us/archive/microsoft-edge/legacy/developer/

Reply:
maybe need to add .md on all such lines.  not sure better way to test than the above.
@captainbrosset 
deviations are bad, consistency in this file is good/helpful.

Reply:
pushed commit to add .md to the above line. pls test @captainbrosset 

the present Comment thread is Outdated.